docs/wiki/update-alz-custom-policies-to-latest.md

```diff
@@ -0,0 +1,280 @@
+- Once all policy assignments are deleted, go to the Definitions pane, search for the definition. Once found click the ellipsis and choose Delete Policy Definition
+
+  ![alz-custom-policy-def-search](media/2.3.update-alz-custom-policy-search.png)
+
+> Important: Record the **Definition location** of the Policy Definition as it will be used in (`$policyDefinitionLocation`) the script below.
+
+- To update to the latest version of the definition, we will use the policy definition templates available in https://github.com/Azure/Enterprise-Scale/tree/main/src/resources/Microsoft.Authorization/policyDefinitions as described in the following.
+
+- Go to https://portal.azure.com
+- Start an Azure Cloud Shell with PowerShell engine
+- Execute the following PowerShell script ([disclaimer](https://github.com/Azure/Enterprise-Scale/blob/main/SUPPORT.md)) for each ALZ custom policy definition:
+  - Before executing the following PowerShell script, update the first two variables:
+    - `$policyDefinitionName`
+    - `$policyDefinitionLocation`
+
+  ```posh
+  $policyDefinitionName = "Deploy-Diagnostics-WVDHostPools" # <-- Replace with policy definition name found earlier
+  $policyDefinitionLocation = "MTB" # <-- Replace with Definition location found earlier
+  $policyDefinitionPath = "./$($policyDefinitionName).json"
+  Invoke-WebRequest -Uri "https://raw.githubusercontent.com/Azure/Enterprise-Scale/main/src/resources/Microsoft.Authorization/policyDefinitions/$($policyDefinitionName).json" -OutFile $policyDefinitionPath
+  $policyDef = Get-Content $policyDefinitionPath | ConvertFrom-Json -Depth 100
+  $policyName = $policyDef.name
+  $displayName = $policyDef.properties.displayName
+  $description = $policyDef.properties.description
+  $mode = $policyDef.properties.mode
+  $metadata = $policyDef.properties.metadata | ConvertTo-Json -Depth 100
+  $parameters = $policyDef.properties.parameters | ConvertTo-Json -Depth 100
+  $policyRule = $policyDef.properties.policyRule | ConvertTo-Json -Depth 100
+  $policyRule = $policyRule.Replace('[[', '[')
+  New-AzPolicyDefinition -Name $policyName -DisplayName $displayname -Description $description -Policy $policyRule -Mode $mode -Metadata $metadata -Parameter $parameters -ManagementGroupName $policyDefinitionLocation
+  ```
+
+> Note that if you decide on another approach from the script above, there are a number of double brackets ('[[') in the file. These need to be replaced with single brackets before the policy set definition is valid syntax.
+
+- To assign *Deploy Diagnostic Settings for AVD Host Pools to Log Analytics workspace* policy, search for that policy definition. Once found click the ellipsis and choose Assign
+
+  ![alz-custom-policy-def-search](media/2.4.update-alz-custom-policy-search.png)
+
+> Note how the display name changed from WVD to AVD.
+
+- Set relevant parameters which were captured earlier.
+
+### Updating ALZ custom policies in ALZ custom policy initiative to newer ALZ custom policies
```

Comment:
I'm missing the step here where newer versions of policies are implemented, could potentially be a reference to the above two sections